gcs/conn.go

```diff
@@ -38,23 +38,25 @@ const (
 	Scope_ReadWrite   = storagev1.DevstorageReadWriteScope
 )
 
+// OpenBucketOptions contains options accepted by Conn.OpenBucket.
+type OpenBucketOptions struct {
+	// Name is the name of the bucket to be opened.
+	Name string
+
+	// BillingProject specifies the project to be billed when making requests.
```

Comment:
"If empty, the option is disabled."

Reply:
The option doesn't really enable anything but is required for a certain types of buckets. This is more explicit in recent commit.